assessments/projects/fluentd/self-assessment.md

```diff
@@ -0,0 +1,223 @@
+
+### Ecosystem
+Fluentd is the default standard to solve Logging in containerized environments, for example, both Docker and Kubernetes has integrated Fluentd. After joining CNCF, Fluentd has formally became a part of a cloud-native stack.
+
+## Security issue resolution
+
+* Responsible Disclosures Process.    
+  - Security vulnerabilites are to be reported at https://github.com/fluent/fluentd/security/advisories, as stated in their [security policy](https://github.com/fluent/fluentd/blob/master/SECURITY.md)
+* Incident Response.  
+  -  Fluentd is trying to follow supply chain security using [DCO](https://probot.github.io/apps/dco/)  
+    [(Supply chain security)](https://github.com/cncf/tag-security/blob/main/supply-chain-security/supply-chain-security-paper/CNCF_SSCP_v1.pdf)  
+  - Because Fluentd is built on top of the Ruby Ecosystems, they mus also check the licenses of dependent gems.
+
+## Appendix
+
+* Known Issues Over Time  
+  There are no documented vulnerabilities for Fluentd  
+
+* CII Best Practices
+  The Fluentd project has adopted the CNCF Code of Conduct,
+  and earned a [Core Infrastructure Initiative Best Practices Badge](https://www.bestpractices.dev/projects/1189#reporting)
+  in  August 2017 
+* Case Studies. Many companies have adopted Fluentd. [List of adopters](https://www.fluentd.org/testimonials)     
+  Here is one such case study:  
+  * LINE- LINE Corporation is known for its messenger app and the various services on its platform,
+    and is faced with the challenge of collecting, storing and analyzing massive log data everyday.
+    The corporation originally used a classic Hadoop setup: They used the Scribe log collector to collect everything into Hadoop
+    and run batch jobs on Hadoop to process the logs. However, there was still room for improvement, which Fluentd provided.
+     * Fluentd is used as the primary data stream processor against terabytes of data everyday.
+     * Using Fluentd, LINE developed many custom plugins, allowing for more flexible data processing
+     * One of LINE's engineers were able to build a schemaless SQL stream processing engine on top of Fluentd.
+[Full details on LINE case study](https://www.fluentd.org/casestudy/line)
+
+* Related Projects / Vendors  
+Fluentd is unique in its use of a [Unified Logging Layer](https://www.fluentd.org/blog/unified-logging-layer) in order to reduce the complexity of backend data parsing.
+Related projects:  
+     * Fluentd, combined with Elastisearch and Kibana, is a free alternative to Splunk. ([Details](https://docs.fluentd.org/how-to-guides/free-alternative-to-splunk-by-fluentd))
+     * Fluentd can be used in conjunction with Norikra to create a SQL-based realtime complex event processing platform. ([Details](https://docs.fluentd.org/how-to-guides/cep-norikra))
+     * Fluentd can be used with Amazon Kinesis to aggregate semi-structured logs in real-time ([Details](https://docs.fluentd.org/how-to-guides/kinesis-stream))
```

Comment:
Was a threat model part of the work done by your team? 
If yes, this is the place to include them.

Reply:
No. A threat model was not conducted by our team.